Gemfile

```diff
@@ -0,0 +1,6 @@
+source 'https://rubygems.org'
```

Comment:
Please remove

Reply:
removing this would give `Your Gemfile has no gem server sources.` could you share context why we need to remove this